ddtrace/tracer/option.go

```diff
@@ -48,6 +53,11 @@ func defaults(c *config) {
 	c.serviceName = filepath.Base(os.Args[0])
 	c.sampler = NewAllSampler()
 	c.agentAddr = defaultAddress
+
+	report := os.Getenv("DD_TRACE_REPORT_HOSTNAME")
+	if b, err := strconv.ParseBool(report); err == nil && b {
```

Comment:
Shouldn't we account for other values like `yes`? Also, why not simply:
```go
if os.Getenv("DD_TRACE_REPORT_HOSTNAME") == "true" {
```
?

Reply:
I prefer the simpler suggestion. Will change.